README.md

```diff
@@ -4,326 +4,117 @@
+- a wide set of **reference models** -- from retrieval baselines to Transformers.
+- seamless **integration of [Amazon Mechanical Turk](https://www.mturk.com/mturk/welcome)** for data collection and human evaluation
+- **integration with [Facebook Messenger](http://www.parl.ai/docs/tutorial_messenger.html)** to connect agents with humans in a chat interface
 
 ParlAI is described in the following paper:
-[“ParlAI: A Dialog Research Software Platform", arXiv:1705.06476](https://arxiv.org/abs/1705.06476).
-
-See the [news page](https://github.com/facebookresearch/ParlAI/blob/master/NEWS.md) for the latest additions & updates, and the website [http://parl.ai](http://parl.ai) for further docs.
-
-## Goals
+[“ParlAI: A Dialog Research Software Platform", arXiv:1705.06476](https://arxiv.org/abs/1705.06476). See the [news page](https://github.com/facebookresearch/ParlAI/blob/master/NEWS.md) for the latest additions & updates.
 
-Unified framework for evaluation of dialogue models
+## Installing ParlAI
 
-- downloads tasks/datasets on demand and provides the same simple interface to them
-- unifies dataset input and evaluation frameworks/metrics
-- `agents/` directory encourages researchers to submit their training code to the repository to share with others
-- aids reproducibility
+ParlAI currently requires Python3. Dependencies of the core modules are listed in requirement.txt. Some models included (in parlai/agents) have additional requirements.
 
-End goal is general dialogue, which includes many different skills
+Run the following commands to clone the repository and install ParlAI:
 
-- seamlessly combines simulated and real language tasks
-- encourages multi-task model development & evaluation
-- helps to reduce overfitting of models to specific datasets         
+```bash
+git clone https://github.com/facebookresearch/ParlAI.git ~/ParlAI
+cd ~/ParlAI; python setup.py develop
+```
 
-End goal is real dialogue with people
+This will link the cloned directory to your site-packages.
 
-- train and evaluate on live dialogue with humans via Mechanical Turk or Messenger
-- easy setup for connecting turkers with your dialogue agent
-- allow to compare different research groups turk experiments
+This is the recommended installation procedure, as it provides ready access to the examples and allows you to modify anything you might need. This is especially useful if you if you want to submit another task to the repository.
 
-Set of datasets to bootstrap a working dialogue model for human interaction
+All needed data will be downloaded to ~/ParlAI/data, and any non-data files (such as the MemNN code) if requested will be downloaded to ~/ParlAI/downloads. If you need to clear out the space used by these files, you can safely delete these directories and any files needed will be downloaded again.
```

Comment:
What is the MemNN code? Perhaps this should be a link for reference? The qualification here is nice, but a way to get more context for beginners would be nice.

Reply:
I also find it easier to read when file paths are wrapped in code blocks, e.g. `~/ParlAI/downloads`... I notice this in some places like here:

https://github.com/facebookresearch/ParlAI/pull/1748/files#diff-04c6e90faac2675aa89e2176d2eec7d8R89

Maybe we can do it everywhere.